### DISCUSSION TITLE
unknown_11  in ebsd.prop

### DISCUSSION THREAD

**Discussion by talkingbird19** [mtex-toolbox/mtex#1573] [Ask Anything] [answered]
hi, dear all

I am currently using mtex-5.8.1. I tried to plot the band contrast,

    plot(ebsd,ebsd.bc)
    colormap gray % make the image grayscale
    mtexColorbar

However, it said that ebsd.bc is an unknown property. Then, I used " ebsd.prop". The answer was,

    confidenceindex: [429015×1 double]
                fit: [429015×1 double]
       imagequality: [429015×1 double]
          semsignal: [429015×1 double]
         unknown_11: [429015×1 double]
         unknown_12: [429015×1 double]
         unknown_13: [429015×1 double]
         unknown_14: [429015×1 double]
                  x: [429015×1 double]
                  y: [429015×1 double]

There are 4 unknown items and I think one of them is bc. Could you kindly help me to solve this problem? The data file is in ".osc" formate.

**Comment by lfgmorales** [comment, marked as answer]
Hi

You have an edax file, and because of that you don’t have band contrast (whic is a property of oxford files). The closest you is the imagequality property, that should do the job

Cheers

Luiz



Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 26 Sep 2022, at 05:19, talkingbird19 ***@***.***> wrote:

﻿

hi, dear all

I am currently using mtex-5.8.1. I tried to plot the band contrast,

plot(ebsd,ebsd.bc)
colormap gray % make the image grayscale
mtexColorbar


However, it said that ebsd.bc is an unknown property. Then, I used " ebsd.prop". The answer was,

confidenceindex: [429015×1 double]
            fit: [429015×1 double]
   imagequality: [429015×1 double]
      semsignal: [429015×1 double]
     unknown_11: [429015×1 double]
     unknown_12: [429015×1 double]
     unknown_13: [429015×1 double]
     unknown_14: [429015×1 double]
              x: [429015×1 double]
              y: [429015×1 double]


There are 4 unknown items and I think one of them is bc. Could you kindly help me to solve this problem? The data file is in ".osc" formate.

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1573>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6TT3ATNBF3LQ4ZRZ3WAEI2BANCNFSM6AAAAAAQVMFVWY>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1573", "url": "https://github.com/mtex-toolbox/mtex/discussions/1573", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by talkingbird19** [reply]
Thank you for your kind help. I got it.

**Reply by talkingbird19** [reply]
hi

I tried to plot by

      plot(ebsd,log(ebsd.prop.iq),'figSize','large')
      mtexColorMap black2white

However, it still says ‘Unrecognized Variable Name “iq”’

Best wishes

**Reply by hakonanes** [reply]
I believe that's because your property is named "imagequality" as per your output above.

**Reply by talkingbird19** [reply]
> I believe that's because your property is named "imagequality" as per your output above.

Smart !!!

Thank youl and best wishes

**Comment by lfgmorales** [comment]
Hi

Correct is plot(ebsd,ebsd.iq)

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 26 Sep 2022, at 13:01, talkingbird19 ***@***.***> wrote:

﻿

hi

I tried to plot by

  plot(ebsd,log(ebsd.prop.iq),'figSize','large')
  mtexColorMap black2white


However, it still says ‘Unrecognized Variable Name “iq”’

Best wishes

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1573#discussioncomment-3732961>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN3IAMB342UDJVD3ZGLWAF7AZANCNFSM6AAAAAAQVMFVWY>.
You are receiving this because you commented.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1573#discussioncomment-3732961", "url": "https://github.com/mtex-toolbox/mtex/discussions/1573#discussioncomment-3732961", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by talkingbird19** [reply]
Thank you for your kind reply.

Best wishes.
